### DISCUSSION TITLE
Watt Offset / Watt Gain & Zwift

### DISCUSSION THREAD

**Discussion by rjlee** [cagnulein/qdomyos-zwift#2257] [Q&A]
I've the app setup successfully on my S22i.  I've tried changing 'Watt Offset' and 'Watt Gain' to 100 & 10 respectively and then restarted the QZ app, however the Watts reported in Zwift still match those reported in iFit manual mode i.e. the two settings don't seem to change the Watts reported in Zwift - is this expected ?

**Comment by cagnulein** [comment]
For sure watt gain can't be changed to values like 10 or 100. Watt gain is a multiplier, watt offset is an additional value instead. What are you trying to do? Usually  nordictrack bikes are reporting wattage greater than reality so I suggest a watt gain of 0.8 to reduce it
